html/profilePage.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
```

Comment:
@RamtinMoshtagh The file path does not meet the expectations. Here are the expectations:

- Authentication page (/index.html)
- Feed Page (/feed/index.html)
- Profile page (/profile/index.html)

What this means is that:

- You should have the `index.html` in the root folder for the auth page.
- You should have the `index.html` in a feed folder for the feed page.
- The `index.html` file should be in a profile folder for the profile page.

See the screenshot below:
![Uploading Project-structure-css-fr.PNG…]()

Reply:
yes, fixed it